src/main/java/net/sf/jabref/bibtex/EntrySorter.java

```diff
@@ -127,18 +117,14 @@ public void databaseChanged(DatabaseChangeEvent e) {
                 break;
             case CHANGED_ENTRY:
                 // Entry changed. Resort list:
-                //Collections.sort(set, comp);
```

Comment:
I interpreted the first line as a comment and the second as a comment-out. :-) But reading it, I see your point. Will fix.